client/src/components/PacmanGame/gameCore.js

```diff
@@ -1,19 +1,13 @@
-export const initGridState = (rows, cols) => {
-  // const column = new Array(cols).fill(0);
-  // return new Array(rows).fill(column);
-  return Array(rows).fill().map(() => Array(cols).fill(0))
-};
-
-
-
-
+export // const column = new Array(cols).fill(0);
+// return new Array(rows).fill(column);
+const initGridState = (rows, cols) => Array(rows).fill().map(() => Array(cols).fill(0));
 export const getFoods = () => [[15, 9], [14, 9], [13, 9], [12, 9], [11, 9], [10, 9]];
 
-const wallGenerator = numberofCells => {
+const wallGenerator = (numberofCells) => {
   const wall = [];
   const int = num => parseInt(num, 10);
 
-  for (let col = 0; col < numberofCells; col++) {
+  for (let col = 0; col < numberofCells; col += 1) {
     const topRow = [col, 0];
```

Comment:
will it be a row or a cell?

Reply:
It should be `topRowCell`. Changed for all row variables under `wallGenerator`